src/module/service.cpp

```diff
@@ -24,7 +24,7 @@
 #include <boost/none.hpp>
 #include <common/utils.hpp>
 #include <components/component_base.hpp>
-#include <components/generic.hpp>
+#include <components/generic/generic.hpp>
```

Comment:
do we still need the generic import here?

Reply:
nope! thanks for spotting